alibi/datasets.py

```diff
@@ -29,6 +31,52 @@
 MOVIESENTIMENT_URLS = ['https://storage.googleapis.com/seldon-datasets/sentence_polarity_v1/rt-polaritydata.tar.gz',
                        'http://www.cs.cornell.edu/People/pabo/movie-review-data/rt-polaritydata.tar.gz']
 
+IMAGENET_URLS = ['https://storage.googleapis.com/seldon-datasets/imagenet10/imagenet10.joblib']
```

Comment:
Why is this in a `.joblib` format instead of a compressed archive `tar.gz`? `.joblib` is basically a version of `pickle` by `sklearn` and is not portable between Python versions and outside Python.

Reply:
Clarified offline: the file contains not just images but also dictionaries of labels so putting them into a pickle/joblib container was the easiest.

However, please add a `TODO` here to store the images and the dictionaries (can be `json`) in portable formats in a follow-up as the `joblib` file is likely to go stale.